pkgs/applications/networking/instant-messengers/qq/update.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl nix-prefetch-git jq
```

Comment:
```suggestion
#!nix-shell -i bash --pure -p git nix curl nix-prefetch-git jq
```

Reply:
I'm not sure if `--pure` will shadow `UPDATE_NIX_ATTR_PATH`, but git and nix are at least needed.